operations/update.go

```diff
@@ -39,12 +40,17 @@ func Update() cli.Command {
 				Name:  joinFlagNames(forceFlagName, "f"),
 				Usage: "download a new CLI even if the current CLI is not out of date",
 			},
+			cli.BoolFlag{
+				Name:  joinFlagNames(autoUpdateFlagName, "a"),
+				Usage: "setup automatic installations of a new CLI if the current CLI is out of date",
+			},
 		},
 		Before: setPlainLogger,
 		Action: func(c *cli.Context) error {
 			confPath := c.Parent().String(confFlagName)
 			doInstall := c.Bool(installFlagName)
 			forceUpdate := c.Bool(forceFlagName)
+			autoUpdate := c.Bool(autoUpdateFlagName)
```

Comment:
If autoUpdate isn't setup, should we print to the user that -a is an option so that they know about it? (Most people will never look at this help page)